src/start.ts

```diff
@@ -5,7 +5,7 @@ import * as node from "@api3/airnode-node";
 import * as adapter from "@api3/airnode-adapter";
 import * as protocol from "@api3/airnode-protocol";
 import * as abi from "@api3/airnode-abi";
-import { flatMap, isEmpty, isNil, map, merge } from "lodash";
+import { flatMap, groupBy, isEmpty, isNil, map, merge } from "lodash";
```

Comment:
Would prefer these to import separately as `import flatMap from 'lodash/flatMap';` since this is importing the entire lodash library currently (which is big). I guess it's probably not a major issue for server side applications though

Reply:
oh nice! Anything that makes the final zip file smaller helps when deploying to aws lambda.